src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -878,6 +920,35 @@ public static String getHoverStringData(RemoteRobot remoteRobot) {
         return popupString.toString();
     }
 
+    /**
+     * Opens the quickfix menu popup and chooses the
+     * approriate fix according to the quickfix substring
+     *
+     * @param remoteRobot the remote robot instance
+     * @param quickfixChooserString the text to find in the quick fix menu
+     */
+    public static void chooseQuickFix(RemoteRobot remoteRobot, String quickfixChooserString) {
+
+        // first trigger the quickfix popup by using the keyboard
+        Keyboard keyboard = new Keyboard(remoteRobot);
+        keyboard.hotKey(VK_ALT, VK_ENTER);
+
+        // get the text from the quickfix popup
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofMinutes(2));
```

Comment:
a 2 minute timeout seems extensive for the quickfix popup to show up - is the framework that slow sometimes?

Reply:
just for clarity - this is a standard timeout value we have been using from the start for getting the ProjectFrame - ir the whole idea - then i go on to get the quickfix popup within the project frame. It is made use of this way throughout the tests with that 2 min timeout.

Having said that i can look into shortening that timeframe - maybe we didn't understand something about the projectframe at the outset and it can indeed be shortened.